Documentation/concepts/matching.md

```diff
@@ -8,6 +8,9 @@ The matcher API is designed to be called often and will always provide the most
 
 See our [api](../howto/api.md) guide to learn how to view our api specification and work with the Matcher api.
 
+The CRDA remote matcher is responsible for fetching vulnerabilities from Red Hat CRDA. CRDA has the best in industry vulnerability data powered by Snyk.
+This matcher by default  serves 100 requests per minute. Each request can have up to 10 packages for optimum performance.The CRDA matcher comes with a configurable option and we can accommodate a much higher rate of requests. Please fill out [CRDA-Request-Form](https://docs.google.com/forms/d/e/1FAIpQLSfJ1aoOfi0tQ6_o0srEbYL-cSwzo_MyDJQnwcFG9IYVlNXJrA/viewform?usp=sf_link)to avail the configuration details.This can then be used in the clair's config file.
+
```

Comment:
* IMO this deserves a subheader
* Can you add something about why this matcher is different from the normal matching process that refers to Clair's internal DB to surface pre-fetched vulnerabilities? I feel like that could really help someone debugging (i.e. why do I see this vulnerability in my report but I can't find it in my DB?).
* There's a double space in there and a couple of spaces missing after `.`s. 
* Not a part of this PR directly but the form also has some spelling errors: `A breif desciption of use case.`

Reply:
ACK. I will modify accordingly.

Reply:
@crozzy Updated it. Decided to create a header  `remote matching` as there is no documentation on that here.